From: Simon McVittie <smcv@debian.org>
Date: Mon, 12 Oct 2020 11:42:33 +0100
Subject: Partially revert "fontchooser: Add user filtering"

Add some #if 0 to stub out more code paths that need Pango 1.47.

This partially reverts commit 09604eb3eb3928429cceeeb235eb4ff3a0bd5169.

Forwarded: not-needed
---
 gtk/gtkfontchooserwidget.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/gtk/gtkfontchooserwidget.c b/gtk/gtkfontchooserwidget.c
index 293e64e..3031cd1 100644
--- a/gtk/gtkfontchooserwidget.c
+++ b/gtk/gtkfontchooserwidget.c
@@ -361,7 +361,7 @@ user_filter_cb (gpointer item,
       !pango_font_family_is_monospace (family))
     return FALSE;
 
-#ifdef HAVE_PANGOFT
+#if 0
   if (self->filter_by_language &&
       self->filter_language)
     {
@@ -892,7 +892,7 @@ axis_free (gpointer v)
   g_free (a);
 }
 
-#ifdef HAVE_PANGOFT
+#if 0
 static void
 select_added (GListModel *model,
               guint       position,
@@ -989,7 +989,7 @@ add_to_fontlist (GtkWidget     *widget,
 
   n = gtk_slice_list_model_get_size (model);
 
-#ifdef HAVE_PANGOFT
+#if 0
   for (i = n; i < n + 10; i++)
     {
       gpointer item = g_list_model_get_item (child_model, i);
@@ -1035,7 +1035,7 @@ update_fontlist (GtkFontChooserWidget *self)
   g_object_unref (model);
 }
 
-#ifdef HAVE_PANGOFT
+#if 0
 static void
 setup_lang_item (GtkSignalListItemFactory *factory,
                  gpointer                  item,
@@ -1166,7 +1166,7 @@ gtk_font_chooser_widget_init (GtkFontChooserWidget *self)
 
   gtk_custom_filter_set_filter_func (self->user_filter, user_filter_cb, self, NULL);
 
-#ifdef HAVE_PANGOFT
+#if 0
   setup_language_list (self);
 #else
   gtk_widget_hide (GTK_WIDGET (self->language_button));
